node/node.go

```diff
@@ -773,9 +773,9 @@ func (node *Node) addNewlyMinedBlock(ctx context.Context, b *types.Block) {
 	}
 }
 
-// miningAddress returns the address of the mining actor mining on behalf of
+// MiningAddress returns the address of the mining actor mining on behalf of
 // the node.
-func (node *Node) miningAddress() (address.Address, error) {
+func (node *Node) MiningAddress() (address.Address, error) {
```

Comment:
so this is just exported for testing, I take it?

Reply:
Used here as a command:
https://github.com/filecoin-project/go-filecoin/pull/3169/files#diff-642261538295948b5c2a6d350214f8e6R31

Used here: https://github.com/filecoin-project/go-filecoin/pull/3169/files#diff-642261538295948b5c2a6d350214f8e6R89

Reply:
ah thanks. I looked for it and didn't see it.